backend/development.ini.sample

```diff
@@ -1,267 +1,373 @@
-# file depot storage
-depot_storage_name = tracim
-depot_storage_dir = %(here)s/depot/
-
-# Backend API config
-# This is needed for some feature like reply by email
-api.key =
-# base url of the backend API, default to same as website.base_url
-# api.base_url = http://localhost:6543
-# title of website, used in email
-website.title = TRACIM
-# The following base_url is used for links and icons
-# integrated in the email notifcations
-website.base_url = http://localhost:6543
-# If config not provided, it will be extracted from website.base_url
-# website.server_name = localhost
-
-# CORS Special config
+# already added to default list.
+# This parameter is a list of app slug separated by ',' char.
+## Not exhaustive list of available apps:
+# - content/thread -> thread/topic/discussion : allow to discuss about one
+#   subject.
+# - content/file -> allow to upload all type of file like .pdf, .odt, .jpg, etc
+# - content/html-document -> allow to work on a standard html-based document
+#   which is editable through tracim.
+# - content/folder -> allow to put content into folder.
+## example of app.enabled config explicit list config
+; app.enabled = contents/thread,contents/file,contents/html-document,contents/folder
+
+## CORS Special config
 # you can override access control allowed origin here with list of comma separated
 # base_url, by default only website.base_url and api.base_url are allowed
-# cors.access-control-allowed-origin = http://localhost:6543,http://localhost:8090
-
-# Auth type (internal or ldap)
+; cors.access-control-allowed-origin = http://localhost:6543,http://localhost:8090
+
+####
+# STORAGE
+####
+
+### Database ###
+# to set database you need a proper sqlalchemy url, some example with supported
+# database (sqlite, postgresql, mysql/mariadb)
+## sqlite
+; sqlalchemy.url = sqlite:///%(here)s/tracim.sqlite
+## mysql/mariadb
+; sqlalchemy.url = mysql+pymysql://user:password@dbhost:dbport/dbname
+## postgresql
+; sqlalchemy.url = postgresql://user:paswword@dbhost:dbport/dbname?client_encoding=utf8
+# see also for more info:
+# https://docs.sqlalchemy.org/en/latest/core/engines.html#sqlalchemy.create_engine ##
+sqlalchemy.url = %(basic_setup.sqlalchemy_url)s
+
+### File Depot ###
+## file depot name
+depot_storage_name = tracim
+## This is where none database file will be stored
+depot_storage_dir = %(basic_setup.depot_storage_dir)s
+
+### Preview ###
+## preview cache directory
+preview_cache_dir = %(basic_setup.preview_cache_dir)s
+
+### Session ###
+# shortcut for pyramid_beaker specific config
+# see tracim_web app section to see where those shortcut are used
+advanced_setup.session_data_dir = %(basic_setup.sessions_data_root_dir)s/sessions_data
+advanced_setup.session_lock_dir = %(basic_setup.sessions_data_root_dir)s/sessions_lock
+
+####
+# AUTH
+####
+
+## Backend API config:
+# Api key auth is disabled by default,
+# you should set a not-null value to api.key to active it:
+api.key = %(basic_setup.api_key)s
```

Comment:
Just to be sure, we are sure about that: api_key not enabled if empty ?

Reply:
yes. api auth is disabled if api_key is empty.